src/apps/common/submodules/numbers/numbers.js

```diff
@@ -1072,20 +1058,24 @@ define(function(require) {
 			});
 		},
 
-		numbersPaintSpare: function(parent, dataNumbers, callback) {
+		numbersRenderSpare: function(args) {
 			var self = this,
+				dataNumbers = args.dataNumbers,
 				template = $(self.getTemplate({
 					name: 'spare',
 					data: dataNumbers,
 					submodule: 'numbers'
 				}));
 
-			parent
+			args.parent
 				.find('.list-numbers[data-type="spare"]')
 				.empty()
 				.append(template);
 
-			callback && callback();
+			var arrayNumbersSpare = dataNumbers.listAccounts.length ? dataNumbers.listAccounts[0].spareNumbers : [];
```

Comment:
The variable `arrayNumbersSpare` should be declared and assigned at the top of the scope, along with the other variable declarations, according to https://docs.2600hz.com/ui/docs/javascriptStyleGuide/#variables